install.yaml

```diff
@@ -156,7 +156,7 @@ post_install_actions:
     {{ $_ := set $service_def "version" $service_type._1 }}
     #echo {{ $service_def }}
     
-    {{ $supported_db_types := list "postgresql" "mariadb" "mysql" }}
+    {{ $supported_db_types := list "postgresql" "mysql" "oracle-mysql" }}
```

Comment:
Should actually be `mariadb` and `oracle-mysql`, since `$supported_db_types` are the ones we support  from the Platform.sh definition. However this means that there is probably a mistake below.

Reply:
Nevermind, I see that you have fixed it in a later commit